soh/src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -489,6 +513,9 @@ void EnDns_Update(Actor* thisx, GlobalContext* globalCtx) {
 
     this->dustTimer++;
     this->actor.textId = D_809F040C[this->actor.params];
+    if (gSaveContext.n64ddFlag && this->scrubIdentity.isShuffled) {
+        this->actor.textId = 0x9000 + this->dnsItemEntry->itemPrice;
```

Comment:
probably worth throwing a comment in here explaining what's happening with the message table being full of messages for each price